ruby-challenges.rb

```diff
@@ -24,6 +41,17 @@
 
 
 
+def remove_vowel(str)
+  temp_str = str.delete('aeiou')
+  p temp_str.delete('AEIOU')
```

Comment:
These can be combined into one delete as delete will check and remove all characters included in the parenthesis.